### DISCUSSION TITLE
Option to add tags in connectors and connections to help filter them

### DISCUSSION THREAD

**Discussion by Dynnammo** [airbytehq/airbyte#30726] [Connector Ideas and Features]
## Context
As a data engineer, I have about a hundred of different sources and destinations to handle, with a number of connections also quite high

## Frustration
When wanting to have an overview or searching for a source/destination/connection, I have to scroll some time before finding it. Since I'm quite organized, I adapted titles to speed up research, but it drives to quite lenghty names

## Solution
Having a system of **object tagging**. It could be available on all "objects" of Airbyte: source, destination and connection. Linked to a system of search bar with filtering, it would greatly improve navigation for users.

## Appendice
Quite new in this forum, I thoughtfully searched through available subjects but didn't found anything related, but I may have missed it, if so, sorry for the disturbance.

**Comment by marcosmarxm** [comment]
This is an interesting idea @Dynnammo 

@timroes I think this idea relies more on front-end + platform side. Any thoughts?

**Comment by tristan-ki** [comment]
This would be a really great addition! Using tags to organise and track resource ownership has become pretty standard practice across the industry.

[a comment by an automated account is not shown]
